pkg/apis/seedmanagement/v1alpha1/helper/gardenlet.go

```diff
@@ -11,18 +11,9 @@ import (
 
 	gardencorev1beta1 "github.com/gardener/gardener/pkg/apis/core/v1beta1"
 	"github.com/gardener/gardener/pkg/apis/seedmanagement/encoding"
-	seedmanagementv1alpha1 "github.com/gardener/gardener/pkg/apis/seedmanagement/v1alpha1"
```

Comment:
Isn't better to name `pkg/apis/resources/v1alpha1/helper/managedresource.go` instead of `pkg/apis/resources/v1alpha1/helper/resourcemanager.go`?